repository/src/main/java/org/alfresco/repo/domain/permissions/AclDAOImpl.java

```diff
@@ -1636,7 +1636,8 @@ public Acl createLayeredAcl(Long indirectedAcl)
         return changes;
     }
 
-    private static final String RESOURCE_KEY_ACL_CHANGE_SET_ID = "acl.change.set.id";
+    public static final String RESOURCE_KEY_ACL_CHANGE_SET_ID = "acl.change.set.id";

```

Comment:
Why did you make them public?

Reply:
I used RESOURCE_KEY_ACL_CHANGE_SET_COMMIT_TIME_MS in a test in the enterprise repository. In order to don't have similar constants with different modifiers, I used the same. Anyway, especially because they are in two different repositories, I prefer to avoid the coupling in the test, so I replaced the public with private and I'll use the string in the test.